src/extensions/default/InlinePaddingEditor/PaddingEditorTemplate.html

```diff
@@ -0,0 +1,119 @@
+<div tabindex="-1" class="padding-editor">
+    <header>
+        <label>Set padding for</label>
```

Comment:
We need to change all hard-coded strings to use mustache templates and gettext for l10n.